src/libxrpl/protocol/STTx.cpp

```diff
@@ -271,6 +272,10 @@ STTx::checkBatchSign(
 {
     try
     {
+        if (getTxnType() != ttBATCH)
+        {
+            JLOG(debugLog().fatal()) << "STTx::checkBatchSign : not a batch transaction";
```

Comment:
Probably also need to `return {}` here otherwise line 282 will fail

Reply:
fixed: https://github.com/XRPLF/rippled/pull/5060/commits/adb41ee7d389288df72866da233a2ab0ddaa4f72